balboa-agent/src/main/java/com/socrata/balboa/agent/MetricConsumer.java

```diff
@@ -147,7 +157,6 @@ public Record(String entityId, String name, Number value, long timestamp, Metric
             while (true) {
                 Map<String, String> record = grovel(stream);
                 if (record == null) {
-                    log.error("Unable to process records from {}", filePath);
                     break;
```

Comment:
Good question, Looking at it again `grovel` returns null when it reaches the end of the file.

Note that I added this log statement with a previous commit and I deleted it now.  I was under the impression that it meant we have reached the end of a file and this was an acceptable return value from ByteArrayOutputStream.read.  But looking at it now I see that this is infact an error case and could lead to the loss of metrics.  It is strange that there was no previous log message there before.